db-scripts/src/main/resources/migration.sql

```diff
@@ -598,4 +598,8 @@ CREATE TABLE `data_access_tokens` (
     PRIMARY KEY (`TOKEN`),
     FOREIGN KEY (`USERNAME`) REFERENCES `users` (`EMAIL`) ON DELETE CASCADE
 );
-UPDATE `info` SET `DB_SCHEMA_VERSION`="2.9.0";
+
+##version: 2.9.1
+ALTER TABLE `copy_number_seg` MODIFY COLUMN `SEG_ID` BIGINT(20);
+
+UPDATE `info` SET `DB_SCHEMA_VERSION`="2.9.1";
```

Comment:
The python script which applies the migration script reads this file line by line - I recommend having the newline at the end of the final line. It very likely does not matter, but good to have just for safety.

Reply:
@ao508 We need to make some minor model changes before this PR is merged:

segID should be changed to Long here:
[CopyNumberSeg.java](https://github.com/cBioPortal/cbioportal/blob/master/model/src/main/java/org/cbioportal/model/CopyNumberSeg.java#L8)

and here:

[CopyNumberSegmentControllerTest.java](https://github.com/cBioPortal/cbioportal/blob/master/web/src/test/java/org/cbioportal/web/CopyNumberSegmentControllerTest.java#L41)
[CopyNumberSegmentControllerTest.java](https://github.com/cBioPortal/cbioportal/blob/master/web/src/test/java/org/cbioportal/web/CopyNumberSegmentControllerTest.java#L51)

Reply:
resolved!